src/styles/blocks/container.scss

```diff
@@ -0,0 +1,15 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  width: $container-size;
+  margin: 0 auto;
+  padding: 0 $card-padding;
```

Comment:
it isn't a good idea to combine margin and padding for BEM block

Reply:
it's a positioning wrapper, no?